jina/resources/logging.default.yml

```diff
@@ -15,3 +15,9 @@ configs:
     host: # when not given then record it locally, /dev/log on linux /var/run/syslog on mac
     port: # when not given then record it locally,  /dev/log on linux /var/run/syslog on mac
     formatter: PlainFormatter
+  RichHandler:
```

Comment:
I am not sure if the RichHandler could be causing problems

Reply:
The RichHandler is here but I don't use it anymore as it breaks logging in docker and k8s